src/biblenotify/ui/BibleNotify/Views/HomeView.qml

```diff
@@ -9,38 +9,65 @@ Item {
     id: root
 
     property var setTimeView
+    property string verseText
+    property string verseReference
 
     signal sendNotification
 
     ColumnLayout {
         anchors.fill: parent
 
-        Image {
+        ColumnLayout {
             Layout.alignment: Qt.AlignHCenter
-            source: "qrc:/illustration-with_text.svg"
-            antialiasing: true
+
+            BNLabel {
+                Layout.alignment: Qt.AlignHCenter
+                Layout.maximumWidth: 600
+                horizontalAlignment: Text.AlignHCenter
+                font.pixelSize: 30
+                color: "#24292f"
+                text: qsTr(verseText)
+            }
+
+            Item {
+                height: 10
+            }
+
+            BNLabel {
+                Layout.alignment: Qt.AlignHCenter
+                Layout.maximumWidth: root.width - root.anchors.leftMargin - root.anchors.rightMargin
+                horizontalAlignment: Text.AlignHCenter
+                font { pixelSize: 20; italic: true; }
+                color: "#616161"
+                text: qsTr(verseReference)
```

Comment:
Same deal as earlier - no point translating this text. The line should be `text: root.verse[1]`.